google/cloud/firestore_v1/async_query.py

```diff
@@ -231,6 +231,40 @@ def count(
         """
         return AsyncAggregationQuery(self).count(alias=alias)
 
+    def sum(
```

Comment:
similar to https://github.com/googleapis/java-firestore/pull/1387/files#diff-fdd9700384056ce568a3fd41266eb1dfc5487c620bade78cde24dd677d7016b5R83, should we also have a constructor that accepts a `FieldPath`?